src/app/qgsmaptoolidentifyaction.cpp

```diff
@@ -114,6 +114,15 @@ void QgsMapToolIdentifyAction::canvasPressEvent( QgsMapMouseEvent* e )
 
 void QgsMapToolIdentifyAction::canvasReleaseEvent( QgsMapMouseEvent* e )
 {
+  // Store clicked x & y into identify menu.
+  if ( mIdentifyMenu )
+  {
+    QgsPoint point = toMapCoordinates(e->pos());
```

Comment:
@nouanda Travis build is failing because of that line. It should be:

```
QgsPoint point = toMapCoordinates( e->pos() );
```

It would be nice to merge this PR, can you fix that?